src/app/repository/repository.service.ts

```diff
@@ -12,6 +13,28 @@ import TEAMS from '../mocks/teams/teams.json';
   providedIn: 'root',
 })
 export class DataRepository {
+  /**
+   * Method will return array with data for char,
+   * number of elements depends depends on difference last request time and current moment
+   * @param lastRequestDateTime time of last request in millisecond format
+   * @returns Observable with array of chart data objects
+   */
+  getChartData(lastRequestDateTime: number): Observable<ChartData[]> {
+    const dataArr: ChartData[] = [];
+    const now = Date.now();
+
+    const timeDifference = lastRequestDateTime
```

Comment:
timeDifference - is it in seconds / ms or what? and why we have ternary operator there?